src/theory/bv/bv_quick_check.cpp

```diff
@@ -145,7 +145,7 @@ bool BVQuickCheck::collectModelInfo(theory::TheoryModel* model, bool fullModel)
 
 BVQuickCheck::~BVQuickCheck() {
   clearSolver();
-  delete d_bitblaster;
+  d_bitblaster.reset();
```

Comment:
Why explicitly delete this here? If the `reset()` is necessary, maybe comment on why otherwise just omit it and let the `std::unique_ptr` take care of it.

Reply:
Yes, you are right. I thought this is actually in a reset method... Will remove it.